src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/Transcript.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { connect } from 'react-redux';
+
+import {
+  Card, Button, IconButton, Row,
+  Icon,
+} from '@edx/paragon';
+import { Delete } from '@edx/paragon/icons';
+
+import {
+  FormattedMessage,
+  injectIntl,
+} from '@edx/frontend-platform/i18n';
+import { thunkActions } from '../../../../../../data/redux';
+
+import TranscriptActionMenu from './TranscriptActionMenu';
+import LanguageSelector from './LanguageSelector';
+import * as module from './Transcript';
+import messages from './messages';
+
+export const hooks = {
+  state: {
+    inDeleteConfirmation: (args) => React.useState(args),
+  },
+  setUpDeleteConfirmation: () => {
+    const [inDeleteConfirmation, setInDeleteConfirmation] = module.hooks.state.inDeleteConfirmation(false);
+    return {
+      inDeleteConfirmation,
+      launchDeleteConfirmation: () => setInDeleteConfirmation(true),
+      cancelDelete: () => setInDeleteConfirmation(false),
+    };
+  },
+};
+
+export const Transcript = ({
+  index,
+  language,
+  // redux
+  deleteTranscript,
+}) => {
+  const { inDeleteConfirmation, launchDeleteConfirmation, cancelDelete } = module.hooks.setUpDeleteConfirmation();
+  return (
+    <>
+      {inDeleteConfirmation
```

Comment:
After deleting a transcript the card does not go away (e.i. the state is not updating to false). How is the state attached to a specific card?

Reply:
> After deleting a transcript the card does not go away.
Ahh! It in fact does, if there is no card below it, if there is, that one takes the index and keeps the state. I will fix this.